reg/drone/srv/air_data_computer/_.0.1.uavcan

```diff
@@ -0,0 +1,48 @@
+# A generic air data computer (ADC) service.
```

Comment:
ADC generates confusion (with analog-digital converter). Please replace this with something like air data sensor (ADS).

Reply:
An air data sensor is a different type of equipment, so it cannot be used here. I have just replaced all uses of ADC with either "air data computer" (unabbreviated) or just "service".